lib/Jsrt/ChakraCore.h

```diff
@@ -165,6 +165,31 @@ JsGetModuleHostInfo(
 
 #ifdef CHAKRACOREBUILD_
 /// <summary>
+///     Returns metadata relating to the exception that caused the runtime of the current context
+///     to be in the exception state and resets the exception state for that runtime. The metadata
+///     includes a reference to the exception itself.
+/// </summary>
+/// <remarks>
```

Comment:
Can you lay out the returned `metadata` object in `<remarks>`? I've always hated that I've no clue what's in `exception` of [JsGetAndClearException](https://github.com/Microsoft/ChakraCore/wiki/JsGetAndClearException) without looking into source code.